src/components/Product/AddToCard/AddToCart.tsx

```diff
@@ -47,11 +48,14 @@ export default function AddToCartButton({ product }: Readonly<Props>) {
             addProduct={addProduct}
             removeProduct={removeProduct}
           />
-          <Button className={'w-[123px] cursor-default hover:bg-brand-solid'}>
+          <Button className='w-[123px] pointer-events-none cursor-default '>
```

Comment:
why is it a button? what function does the price block perform besides displaying the price?